vms/platformvm/service.md

```diff
@@ -1226,8 +1229,10 @@ platform.getSubnet({
   a permissioned subnet. If the Subnet is a PoS Subnet, then `threshold` will be `0` and `controlKeys`
   will be empty.
 - changes can not be made into the subnet until `locktime` is in the past.
-- `subnetTransformationTxID` is the ID of the transaction that changed the subnet into a elastic one,
-  for when this change was performed.
+- `subnetTransformationTxID` is the ID of the transaction that changed the subnet into an elastic one, if it exists.
```

Comment:
```suggestion
- `subnetTransformationTxID` is the ID of the transaction that changed the subnet into an L1, if it exists.
```

Reply:
realized this is probably only referencing the tx has of `TransformSubnetTx` can this field not be removed from the response since it has been deprecated?

Reply:
I don't think it makes sense to remove this as part of this PR.